tests/functional/adapter/snapshots/fixture_snapshots.py

```diff
@@ -0,0 +1,386 @@
+# updating original fixtures, because SingleStore doesn't support schema concept
+
+create_seed_sql = """
+create table seed (
+    id INTEGER,
+    first_name VARCHAR(50),
+    last_name VARCHAR(50),
+    email VARCHAR(50),
+    gender VARCHAR(50),
+    ip_address VARCHAR(20),
+    updated_at TIMESTAMP
```

Comment:
do we need to use TIMESTAMP here, I'm wondering if it makes sense to change it DATETIME